When I ran this locally (merged this PR with https://github.com/ome/omero-guide-trackmate/pull/4 and run ``repo2docker .``, I am getting an error

<details>

```
Step 41/53 : RUN TIMEFORMAT='time: %3R' bash -c 'time mamba env update -p ${NB_PYTHON_PREFIX} -f "binder/environment.yml" && time mamba clean --all -f -y && mamba list -p ${NB_PYTHON_PREFIX} '
 ---> Running in 7d1d0c026862

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/srv/conda/lib/python3.7/site-packages/conda/exceptions.py", line 1079, in __call__
        return func(*args, **kwargs)
      File "/srv/conda/lib/python3.7/site-packages/conda_env/cli/main.py", line 80, in do_call
        exit_code = getattr(module, func_name)(args, parser)
      File "/srv/conda/lib/python3.7/site-packages/conda_env/cli/main_update.py", line 124, in execute
        result[installer_type] = installer.install(prefix, specs, args, env)
      File "/srv/conda/lib/python3.7/site-packages/mamba/mamba_env.py", line 40, in mamba_install
        index = get_index(tuple(_channel_priority_map.keys()), prepend=False)
      File "/srv/conda/lib/python3.7/site-packages/mamba/utils.py", line 61, in get_index
        api.create_cache_dir(), api.cache_fn_url(full_url)
    RuntimeError: No space left on device: '/srv/conda/pkgs/cache'

`$ /srv/conda/bin/mamba update -p /srv/conda/envs/notebook -f binder/environment.yml`

  environment variables:
                 CIO_TEST=<not set>
  CONDA_AUTO_UPDATE_CONDA=false
        CONDA_DEFAULT_ENV=/srv/conda/envs/notebook
                CONDA_DIR=/srv/conda
               CONDA_ROOT=/srv/conda
                     PATH=/home/pwalczysko/.local/bin:/home/pwalczysko/.local/bin:/srv/conda/env
                          s/notebook/bin:/srv/conda/bin:/srv/npm/bin:/usr/local/sbin:/usr/local/
                          bin:/usr/sbin:/usr/bin:/sbin:/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : None
       user config file : /home/pwalczysko/.condarc
 populated config files : /srv/conda/.condarc
          conda version : 4.8.2
    conda-build version : not installed
         python version : 3.7.6.final.0
       virtual packages : __glibc=2.27
       base environment : /srv/conda  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /srv/conda/pkgs
                          /home/pwalczysko/.conda/pkgs
       envs directories : /srv/conda/envs
                          /home/pwalczysko/.conda/envs
               platform : linux-64
             user-agent : conda/4.8.2 requests/2.22.0 CPython/3.7.6 Linux/5.10.25-linuxkit ubuntu/18.04.5 glibc/2.27
                UID:GID : 148084480:148084480
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.

time: 0.998
Removing intermediate container 7d1d0c026862
The command '/bin/sh -c TIMEFORMAT='time: %3R' bash -c 'time mamba env update -p ${NB_PYTHON_PREFIX} -f "binder/environment.yml" && time mamba clean --all -f -y && mamba list -p ${NB_PYTHON_PREFIX} '' returned a non-zero code: 1(repo2docker
```

</details>

--- a/notebooks/tracking.ipynb
+++ b/notebooks/tracking.ipynb
@@ -146,7 +146,7 @@
     "PORT = 443\n",
     "\n",
     "image_id = ga['ImageID'].toLong()\n",
-    "group_id = -1\"
+    "group_id = -1"
    ]
   },
   {

